docs/release-notes/8.13.asciidoc

```diff
@@ -1,6 +1,24 @@
 [[release-notes-header-8.13.0]]
 == 8.13
 
+[discrete]
+[[release-notes-8.13.3]]
+=== 8.13.3
+
+[discrete]
+[[enhancements-8.13.3]]
+==== Enhancements
+
+* Ensures that {elastic-defend} can’t be installed on versions earlier than Windows 10 or Server 2016.
+
+[discrete]
+[[bug-fixes-8.13.3]]
+==== Bug fixes
+* Fixes a bug that causes a warning to display after you add an Endpoint exception to a rule ({kibana-pull}180800[#180800]).
```

Comment:
as per https://github.com/elastic/kibana/issues/160851#issuecomment-1851596652, it was happening for rule exceptions too

Reply:
Thanks, @vitaliidm - how's this:
```suggestion
* Fixes a bug that causes a warning to display after you add any type of exception to a rule ({kibana-pull}180800[#180800]).
```

Reply:
Added in [afcab61](https://github.com/elastic/security-docs/pull/5141/commits/afcab61ecf57c1ca3116fad5ea71d612b37a9b0b).